src/lt/plugins/clojure.cljs

```diff
@@ -870,3 +877,24 @@
                 :tags #{:clojure.lang})
 
 (def clj-lang (object/create ::langs.clj))
+
+(def browser-paths
+  "Relative paths to search for when connecting to a clojurescript browser."
+  ["index.html" "resources/public/index.html" "public/index.html" "dev-resources/public/index.html"])
+
+(defn find-default-html-file
+  "Searches browser-paths for first relative path to exist and returns it."
+  [ed]
+  (let [project-dir (files/parent (files/walk-up-find (get-in @ed [:info :path]) "project.clj"))]
```

Comment:
Agreed but boot only works for browser external connections and this currently only supports browser (internal) connections. I'll add boot if we ever add browser external support for 'ClojureScript Browser'